local-build/org.eclipse.swt.fragments.localbuild/META-INF/p2.inf

```diff
@@ -38,3 +38,13 @@ requires.8.namespace = org.eclipse.equinox.p2.iu
 requires.8.name = org.eclipse.swt.gtk.linux.riscv64
 requires.8.range = 0.0.0
 requires.8.filter = (&(osgi.os=linux)(osgi.ws=gtk)(osgi.arch=riscv64))
+
```

Comment:
This part can not go in unless actual binaries are produced and pushed.

Reply:
Ditto, per my reply above